### PR TITLE
Fixes with MOAB 4.7 release

### PR DESCRIPTION
**Pull request by rajeeja**
Changing website, link and sha.

### CODE REVIEW

**Comment by dpo** [comment]
@BrewTestBot test this please

**Comment by dpo** [comment]
I think this needs to be rebased against `origin/master`.

File: moab.rb [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
 
-  depends_on :autoconf
-  depends_on :automake
-  depends_on :libtool
   depends_on 'netcdf'
   depends_on 'hdf5'
-  depends_on :fortran if build.with? 'check'
+  depends_on 'gcc'
```
Comment (iml) [inline moab.rb]: Don't depend on `gcc`, just `:fortran`. If it really doesn't build with clang, the `fails_with :clang` DSL should handle that. If the issue is libc++ incompatibility, pass `ENV.libstdcxx` to the install method.

File: moab.rb [hunk 2]
```diff
@@ -1,27 +1,30 @@
-require 'formula'
+require "formula"
 
 class Moab < Formula
-  homepage 'https://trac.mcs.anl.gov/projects/ITAPS/wiki/MOAB'
-  url 'https://bitbucket.org/fathomteam/moab/get/4.6.2.tar.gz'
-  sha1 'cecd30aaf9bdbb58078e06b0097a9144cac63d7f'
+  homepage "http://sigma.mcs.anl.gov"
+  url "http://ftp.mcs.anl.gov/pub/fathom/moab-nightly.tar.gz"
```
Comment (iml) [inline moab.rb]: Need a stable release.

**Comment by iml** [comment]
Since this was submitted on master, you might need to do a hard reset and recommit.

**Pull request closed without merging**